scripts/ase_example.py

```diff
@@ -31,35 +30,42 @@ def run_relaxation(calculator, filename, steps=500):
     slab.center(vacuum=13.0, axis=2)
     slab.set_pbc(True)
     slab.set_calculator(calculator)
-    dyn = BFGS(slab, trajectory=filename)
+    print("### Generating data")
+    dyn = BFGS(slab, trajectory=filename, logfile=None)
     dyn.run(fmax=0.01, steps=steps)
 
 
 if __name__ == "__main__":
 
     # Generate sample training data
+    os.makedirs("../data/data/example/", exist_ok=True)
     run_relaxation(
-        calculator=EMT(), filename="./COCu_emt_relax.traj", steps=200
+            calculator=EMT(), filename="../data/data/example/COCu_emt_relax.traj", steps=200
```

Comment:
Extra space? And should we make it `data/data/...` so that it's consistent with other scripts (assuming it's run from root directory as `python scripts/ase_example.py`)?